index.bs

```diff
@@ -1527,6 +1531,483 @@ longer be considered as sanitized.
 
 A more complete treatment of mXSS can be found in [[MXSS]].
 
+
+# Index
+
+<em>This section is not normative.</em>
+
+## Elements and attributes not included in the default config ##  {#default-disallowed-elements}
+
+While the Sanitizer aims to disallow Cross-Site Scripting attacks, the default
+settings are a bit stricter and do not include elements that may change page
+settings, layout or fetch resources from other origins. The following elements
+can be allowed with a configuration, but the individual reasoning for disallowing
+them by default is given below.
+
+Some elements are disallowed for multiple reasons, the list below does NOT aim to be
+exhaustive. The following list will only mention the most concerning capability as the
+reason for not allowing an element by default. It does not list all of them.
+
+See also: [[#security-considerations]].
+
+: `<meta>`
+:: This element is not allowed, because it can set or modify global document state.
+
+: `<link>`
+:: This element is not allowed, because it can set or modify global document state.
+
+: `<a target>`
```

Comment:
Mixing elements and attributes into one list makes it hard to scan.

Reply:
Good point. Some of the elements were per-element and some were global. Let me try to strike a balance here in a follow-up